Gemfile.lock

```diff
@@ -392,3 +384,6 @@ DEPENDENCIES
   webmock (= 1.17.3)
   whenever (~> 0.9.4)
   zeroclipboard-rails
+
+BUNDLED WITH
+   1.10.6
```

Comment:
because 1.9.4 was the latest version when we rolled out 2.2.2 (2.2.3 is deployed with 1.10.6).

Assuming you're running on the dev VM, this should be all set up for you by puppet.